### ISSUE TITLE
Fix readme and some confusing installation problem

### ISSUE THREAD

**Issue by Trimardio** [widelands/widelands-website#316]
Ok, maybe they aren't THAT confusing, but I guess I'm stupid, and some other stupid people might come after me. So quick list:

- local_urls.py.sample and local_settings.py.sample should be store in mainpage directory, because that's there that they should be copied to

- .gitignore should be updated for the location of these files

- Either the proposed command to copy these file should be copy or it should be highlighted that symlinks only work with absolute paths. 

- readme shouldn't propose bzr anymore as necessary package

I could fix those and propose the pull request, but I wanted your opinions first :)

**Comment by frankystone** [comment]
> * local_urls.py.sample and local_settings.py.sample should be store in mainpage directory, because that's there that they should be copied to
> * .gitignore should be updated for the location of these files

Not sure about this. The intention to have those files in the base directory was: The .py.sample files are not really part of a working installation.
 
> * Either the proposed command to copy these file should be copy or it should be highlighted that symlinks only work with absolute paths.

Can you test a different workflow for symlinking?
`cd mainpage`
`$ ln -s ../local_urls.py.sample local_urls.py`
`$ ln -s ../local_settings.py.sample local_settings.py`

But since i always copied them, i am in favor of exchanging the symlinking part with a copying part in the README.

> * readme shouldn't propose bzr anymore as necessary package

The actual README does not have bzr as needed package. You may have to pull master into your branch?

The only place where bzr is mentioned in the readme is line 140:
https://github.com/widelands/widelands-website/blob/master/README.txt#L140

This should be adjusted, imho. But https://www.widelands.org/wiki/GitPrimer/ is a confusing resource, at least for me.
 
> 
> I could fix those and propose the pull request, but I wanted your opinions first :)

Feel free to do so :-)

**Comment by Trimardio** [comment]
> Not sure about this. The intention to have those files in the base directory was: The .py.sample files are not really part of a working installation.

Ok, we could have made a new folder with samples too. But I guess that's uncessary complexity. So I focused on emphasing the copy INSIDE et directory. If you guys can't check tthe english I'm not sure^^

>Can you test a different workflow for symlinking?
> cd mainpage
> $ ln -s ../local_urls.py.sample local_urls.py
> $ ln -s ../local_settings.py.sample local_settings.py

Nope, tested again, same error. When copy or absolute path it works though!

>The actual README does not have bzr as needed package. You may have to pull master into your branch?

Indeed sorry^^

**Comment by Trimardio** [comment]
I added *.pyc to the gitignore cause I really don't see in which context you would want to commit that? Or is that an error?

**Comment by frankystone** [comment]
Since python3 *.pyc files are placed inside the directorie(s) __pycache__, which are inside the subfolders. So excluding the folders named __pycache__ should be sufficient. Not sure if the current entry in .gitignore will catch them in the subfolders.

Do you have *.pyc files beside *.py files (e.g. mainpage/settings.py and mainpage/settings.pyc)?

**Comment by Trimardio** [comment]
Yes I do, precisely the example you gave.  It only does that inside the mainpage directory now that you say it. 

For the module the pyc files do go inside the __pycache__ directory though. So yes the gitignore is appropriated for those

**Comment by frankystone** [comment]
I've made a short test and i do  not have any *.pyc files in the directory mainpage/.
```
ls -l mainpage/
insgesamt 100
-rw-r--r-- 1 kaputtnik users     0  3. Okt 16:59 __init__.py
drwxr-xr-x 2 kaputtnik users  4096 25. Nov 22:03 __pycache__
-rw-r--r-- 1 kaputtnik users  1678  3. Okt 16:59 admin.py
-rw-r--r-- 1 kaputtnik users   600  3. Okt 16:59 forms.py
-rw-r--r-- 1 kaputtnik users  3329 25. Nov 22:03 local_settings.py
-rw-r--r-- 1 kaputtnik users   788 25. Nov 22:03 local_urls.py
-rw-r--r-- 1 kaputtnik users   462  3. Okt 16:59 mainpage_urls.py
-rw-r--r-- 1 kaputtnik users    57  3. Okt 16:59 models.py
-rw-r--r-- 1 kaputtnik users  2145  3. Okt 16:59 online_users_middleware.py
-rw-r--r-- 1 kaputtnik users 12373  3. Okt 16:59 settings.py
-rw-r--r-- 1 kaputtnik users   674  3. Okt 16:59 sitemap_urls.py
-rw-r--r-- 1 kaputtnik users   339  3. Okt 16:59 sitemaps.py
drwxr-xr-x 5 kaputtnik users  4096  3. Okt 16:59 static
drwxr-xr-x 3 kaputtnik users  4096  3. Okt 16:59 templates
drwxr-xr-x 3 kaputtnik users  4096 25. Nov 22:01 templatetags
-rwxr-xr-x 1 kaputtnik users  1207  3. Okt 16:59 tests.py
-rw-r--r-- 1 kaputtnik users  2871  3. Okt 16:59 urls.py
drwxr-xr-x 2 kaputtnik users  4096  3. Okt 16:59 utest
-rw-r--r-- 1 kaputtnik users  1089  3. Okt 16:59 validators.py
-rw-r--r-- 1 kaputtnik users  6144  3. Okt 16:59 views.py
-rw-r--r-- 1 kaputtnik users  2099  3. Okt 16:59 wl_utils.py
-rw-r--r-- 1 kaputtnik users  1014  3. Okt 16:59 wlwebsite_wsgi.py
```

Please delete all files named  like '*.pyc'

` find . -name  '*.pyc' -delete`

and run `./manage.py runserver` again. If you have *.pyc- files again outside the directories called `__pycache__`, there is something wrong with your virtualenvironment... just  a guess :-)

Sometimes remaining complied python files (\*.pyc) can lead into confusion/malfunction, especially if the original file (\*.py) was deleted/renamed. This behavior had cost me hours of time to investigate in the past :D

**Comment by frankystone** [comment]
Closed via #318
